evap/evaluation/models.py

```diff
@@ -583,7 +583,7 @@ def update_courses(cls):
 
 
 @receiver(post_transition, sender=Course)
-def log_state_transition(sender, instance, name, source, target, **kwargs):
+def log_state_transition(instance, name, source, target, **_kwargs):
```

Comment:
using `**_kwargs` makes copying methods from other places in the code or from the documentation a little bit more work. i think i'd prefer listing `kwargs` in the `dummy-variables-rgx` and keeping the name the same everywhere regardless whether it's actually used in the method or not.

Reply:
by "a little bit more work" you mean adding an underscore? :)

i think there should be good reasons for ignoring parameters, and i don't see why that shouldn't apply to keyword arguments. additionally, in our codebase it's really just the receivers that are not using kwargs.

Reply:
especially i mean thinking about anything in addition to the actual code. it is no problem at all for you and me, but a small one for less experienced contributors. the linter won't explain them what to do here and they might wonder why something they read in the docs doesn't fulfill the requirements. so it will always be a comment in the reviews explaining them what to do.
but as it makes the code more correct i'm on your side that we shouldn't handle them differently, so i'm okay with keeping the proposed solution.